nmap-update/Makefile.in

```diff
@@ -10,7 +10,7 @@ CC = @CC@
 CFLAGS = @CFLAGS@
 CPPFLAGS = @CPPFLAGS@ $(DEFS) -I$(NBASEDIR)
 LDFLAGS = @LDFLAGS@
-LIBS = @LIBS@ $(NBASELIB)
+LIBS = @LIBS@ $(NBASELIB) -lsqlite3 -lz -lsasl2 -liconv -framework CoreFoundation -framework Security
```

Comment:
This Makefile is intended for use on all platforms, so platform-specific stuff like this should either be handled in the configure script or in the `macosx/Makefile`. Check whether it's needed for dynamic linking build by just doing a `./configure && make nmap-update` from the top-level source dir on OS X. If not, then it's specific to our packaging process and should go into the `macosx/Makefile`.